Tests/ApodiniDeployTests/WebServiceStructureExportTests.swift

```diff
@@ -14,6 +14,7 @@ import ApodiniUtils
 @testable import ApodiniDeployBuildSupport
 import DeploymentTargetLocalhostCommon
 import DeploymentTargetAWSLambdaCommon
+import ApodiniDeployTestWebService
```

Comment:
Am I right that the only reason why the `ApodiniDeployTestWebService` got split into `ApodiniDeployTestWebService` library and `ApodiniDeployTestWebServiceExcutable` is for those testing purposes here?

I think the only reason why we have a dedicated deployment test web service is for e.g. the `LocalhostDeploymentProviderTests` aiming at a complete test coverage from compilation to execution.

Now, that all this happens in a single process, can this be simplified? 
For one, I don't think we need to further split the target into two. Just define a simple TestWebService testing against the web service structure retrieval.
Second, can we rewrite the tests such that the compilation step isn't necessary anymore (and therefore the sprat test executable can be removed completely) or did I misunderstand this refactoring?